easybuild/easyblocks/w/wps.py

```diff
@@ -275,8 +288,12 @@ def run_wps_cmd(cmdname, mpi_cmd=True):
                 # GEOGRID
 
                 # setup directories and files
-                for d in os.listdir(os.path.join(tmpdir, "geog")):
-                    os.symlink(os.path.join(tmpdir, "geog", d), os.path.join(tmpdir, d))
+                if mainver < 4:
+                    geog_data_dir = "geog"
+                else:
+                    geog_data_dir = "WPS_GEOG"
+                for d in os.listdir(os.path.join(tmpdir, geog_data_dir)):
+                    os.symlink(os.path.join(tmpdir, geog_data_dir, d), os.path.join(tmpdir, d))
```

Comment:
While touching this, please switch to using the symlink function that is available in easybuild.tools.filetools, and avoid using single-letter variable names outside of list comprehensions (so use ‘dir_name’ rather than ‘d’).